.travis.yml

```diff
@@ -198,71 +198,17 @@ jobs:
       script:
         - bash ./scripts/docker-run-memory-test.sh
 
-    # Valgrind test #1 : first 40 excluding plan_hashagg_optimized* and plan_hashagg_results*
-    - if: (branch = prerelease_test)
-      stage: test
-      name: "Valgrind test #1"
-      before_install:
-      install:
-      after_failure:
-      before_script:
-        # clone valgrind test script
-        - git clone https://$USR:$PASSWORD@bitbucket.org/440-labs/tsdb-dev-tools.git /tmp/tsdb-dev-tools
-        - bash -c 'while true; do echo -e "keepalive\n"; sleep 60; done' & # print line every 60 minutes to avoid Travis timeout
-      script:
-        # TEST_MAX specifies the maximum test # to go up to
-        - cd /tmp/tsdb-dev-tools; USE_REMOTE=true TIMESCALE_DIR=$TRAVIS_BUILD_DIR TEST_MAX=35 EXCLUDE_PATTERN='plan_hashagg_optimized*|plan_hashagg_results*' bash ./test_valgrind.sh
-      after_script:
-        - kill $(jobs -p) # kill job that prints repeatedly
-
-    # Valgrind test #2: tests matching plan_hashagg_optimized*
-    - if: (branch = prerelease_test)
-      stage: test
-      name: "Valgrind test #2"
-      before_install:
-      install:
-      after_failure:
-      before_script:
-        # clone valgrind test script
-        - git clone https://$USR:$PASSWORD@bitbucket.org/440-labs/tsdb-dev-tools.git /tmp/tsdb-dev-tools
-        - bash -c 'while true; do echo -e "keepalive\n"; sleep 60; done' & # print line every 60 minutes to avoid Travis timeout
-      script:
-        # TEST_MAX specifies the maximum test # to go up to
-        - cd /tmp/tsdb-dev-tools; USE_REMOTE=true TIMESCALE_DIR=$TRAVIS_BUILD_DIR INCLUDE_PATTERN='plan_hashagg_optimized*' bash ./test_valgrind.sh
-      after_script:
-        - kill $(jobs -p) # kill job that prints repeatedly
-
-    # Valgrind test #3: tests matching plan_hashagg_results*
-    - if: (branch = prerelease_test)
+    # Sanitizer tests
+    - if: (type = cron) OR (branch = prerelease_test)
       stage: test
-      name: "Valgrind test #3"
+      name: "Sanitizer tests"
```

Comment:
I would make this a bit more descriptive - specify that this is ASAN and UBSAN. That way we don't trick ourselves into thinking this is MSAN too.

Reply:
updated